llpc/tool/amdllpc.cpp

```diff
@@ -343,6 +343,14 @@ cl::opt<bool> GpuRtUseDumped("gpurt-use-dumped", cl::desc("Use the GPURT shader
 
 cl::opt<std::string> GpuRtLibrary("gpurt-library", cl::desc("Use the GPURT shader library from the given file"));
 
+cl::opt<LlpcRaytracingMode>
+LlpcRaytracingModeSetting("llpc-raytracing-mode", cl::init(LlpcRaytracingMode::Legacy),
+                          cl::desc("Override the LLPC raytracing mode"),
+                          cl::values(
+                            clEnumValN(LlpcRaytracingMode::Legacy, "legacy", "Legacy mode"),
+                            clEnumValN(LlpcRaytracingMode::Gpurt2, "continufy", "Legacy RT pipeline with continufy"),
```

Comment:
@flakebi Do we need to handle this enum value properly (preprocessor guard)?

Reply:
This is still an issue.